core/src/main/scala/org/broadinstitute/dsp/workbench/hamm/db/DbReference.scala

```diff
@@ -28,7 +27,8 @@ object DbReference extends HammLogger {
       liquibase.update(new Contexts())
     } catch {
       case e: SQLTimeoutException =>
-        val isCertProblem = Throwables.getRootCause(e).isInstanceOf[SunCertPathBuilderException]
+        val isCertProblem = org.apache.commons.lang3.exception.ExceptionUtils.getRootCause(e).isInstanceOf[SunCertPathBuilderException]
```

Comment:
replacing `com.google.common.base.Throwables` because it comes from `grpc-netty`. And it's easy to get into bin compat issue with grpc, so it's not worth bringing in `grpc-netty` just to use `com.google.common.base.Throwables.getRootCause`

Reply:
thanks!